conda/recipes/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set data = load_file_data("pyproject.toml") %}
+
+package:
+  name: nvjitlink
+  version: {{ data.get("project", {}).get("version", "0.1.0.dev1") }}
+
+source:
+  path: ../../
+
+build:
+  noarch: python
+  script:
+    # This is a workaround for library search path issues with the current
+    # cuda-nvcc package. They should be fixed by
+    # https://github.com/conda-forge/cuda-nvcc-feedstock/pull/24, at which
+    # point we can remove this.
+    - export CMAKE_PREFIX_PATH=${PREFIX}/targets/x86_64-linux/lib:$CMAKE_PREFIX_PATH
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake >=3.26.4
+    - sysroot_{{ target_platform }} 2.17
+    - cuda-version
+  host:
+    - libnvjitlink-dev
+    - libnvjitlink-static
+    - cuda-version
+    - python
+    - pip
+    - scikit-build-core
+  run:
+    - python
+    - numba >=0.57
+    - {{ pin_compatible('cuda-version', max_pin='x', min_pin='x') }}
+
+test:
+  imports:
+    - nvjitlink
+  source_files:
+    - nvjitlink/tests/test_nvjitlink_api.py
+    - nvjitlink/tests/test_nvjitlink.py
+    # Extra files for testing edge cases of libnvjitlink
+    # see test_nvjitlink_api.py for details
+    - nvjitlink/tests/test_device_functions.cubin
+    - nvjitlink/tests/test_device_functions.fatbin
+    - nvjitlink/tests/undefined_extern.cubin
+  requires:
+    - pytest
+  commands: 
```

Looks like the linter [caught some extra whitespace]( https://github.com/rapidsai/pynvjitlink/actions/runs/7493476170/job/20399259698?pr=2#step:6:95 )

```suggestion
  commands:
```